src/Altinn.Apps/AppFrontend/react/shared/src/components/molecules/AltinnMobileTableItem.tsx

```diff
@@ -71,9 +71,9 @@ export default function AltinnMobileTableItem({
     >
       <Table className={classes.table}>
         <TableBody>
-          {items.map((item) => {
+          {items.map((item, i) => {
             return (
-              <TableRow key={item.label}>
+              <TableRow key={i}>
```

Comment:
In general, using indexes for keys is not a good pattern (f.ex https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/ )  - is this change nescessary?

Reply:
`item.Label` comes from our text resources and the correct type is `React.ReactElement` not `string`. It probably is OK as long as no one tries to put html in the title of a group element.

I know indexes are bad, but this did not seem like one of the cases where it is problematic, because the list of child elements in a group should be constant.

Reply:
Ah, I see.. So label is not a great fit here then. Could perhaps use the item.value instead? That should be a string I guess, and would likely be unique.

If that doesnt work, then I agree, indexes could be used here.

Reply:
I considered `.value`, but without checking too well, I assumed the value would be likely to not always be unique and also subject to change based on user input.

Reply:
It suddenly occurred to me how to fix this properly in 06bf98ea0f